packages/node-sdk/src/client.ts

```diff
@@ -987,6 +1053,14 @@ export class BoundBucketClient {
   private readonly _client: BucketClient;
   private readonly _options: ContextWithTracking;
 
+  /**
+   * Creates a new BoundBucketClient.
```

Comment:
this is basically an "internal" constructor. You should be using the `client.bindClient()` method and not this one directly. Can we make it `@internal`?